src/lib.rs

```diff
@@ -1235,6 +1235,12 @@ impl Builder {
         self.options.no_hash_types.insert(arg.into());
         self
     }
+
+    /// Use associated constants for bitfields.
+    pub fn use_associated_consts(mut self, doIt: bool) -> Builder {
```

Comment:
This probably deserves a bit more specific name. And please use `snake_case` for the argument name.